Bottom: b9c19fb4396d0df37bda5f1c4ecb6d52890c3112
Top:    6299f7df8d8e53648f6fd9125579ad6e44a4dae1
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-09-13 15:43:00 -0500

fixup openldap



---
diff --git a/roles/docker-openldap/tasks/main.yml b/roles/docker-openldap/tasks/main.yml
index a62a8f2e..629032ad 100644
--- a/roles/docker-openldap/tasks/main.yml
+++ b/roles/docker-openldap/tasks/main.yml
@@ -151,10 +151,11 @@
       LDAP_HOSTNAME: '{{ openldap_hostname }}'
     hostname: '{{ phpldapadmin_hostname }}'
     image: '{{ phpldapadmin_image_name }}:{{ docker_image_tag }}'
-    link: '{{ openldap_container_name }}:ldap'
     name: '{{ phpldapadmin_container_name }}'
     networks:
       - name: '{{ docker_network_frontend }}'
+        links:
+          - '{{ openldap_container_name }}:ldap'
     ports: '{{ phpldapadmin_port_args }}'
     purge_networks: true
     restart_policy: '{{ docker_restart_policy }}'
